docs/analytics.md

```diff
@@ -112,4 +112,10 @@ You can download and visualize the results:
 ```python
 job.download_results()
 job.plot_results()
-```
\ No newline at end of file
+```
+
+## Troubleshooting
+
+### How many blocks can I add to a workflow?
```

Comment:
To the overview:
"A workflow starts with a data block and is usually followed by one or more processing blocks."